src/lstm/recodebeam.h

```diff
@@ -425,6 +428,11 @@ class TESS_API RecodeBeamSearch {
   bool is_simple_text_;
   // The encoded (class label) of the null/reject character.
   int null_char_;
+  // Variables used in tracking possible diplopia case
```

Comment:
Adding that new block after `bool is_simple_text_` would reduce the class size because it avoids another hole.